MAINTAINERS.md

```diff
@@ -255,7 +255,8 @@ From `origin/master`, begin a branch for syncing Endo.
 
 ```sh
 NOW=`date -u +%Y-%m-%d-%H-%M-%S`
-git checkout -b "$USER-sync-endo-$NOW" origin/master
+git checkout -b "$USER-sync-endo-$NOW" origin/endo-integration-master
+git rebase origin/master
```

Comment:
After all is done, we should end up with `origin/endo-integration-master` to be the same as either the merge of the resulting PR, or to simplify, the head of the sync branch.